mne/io/meas_info.py

```diff
@@ -367,17 +369,45 @@ def _read_dig_points(fname, comments='%'):
     comments : str
         The character used to indicate the start of a comment;
         Default: '%'.
+    unit : 'auto' | 'm' | 'cm' | 'mm'
+        Unit of the digitizer files (hsp and elp). If not 'm', coordinates will
+        be rescaled to 'm'. Default is 'auto', which assumes 'm' for *.hsp and
+        *.elp files and 'mm' for *.txt files, corresponding to the known
+        Polhemus export formats.
 
     Returns
     -------
     dig_points : np.ndarray, shape (n_points, 3)
-        Array of dig points.
+        Array of dig points in [m].
     """
-    dig_points = np.loadtxt(fname, comments=comments, ndmin=2)
+    if unit not in ('auto', 'm', 'mm', 'cm'):
+        raise ValueError('unit must be one of "auto", "m", "mm", or "cm"')
+
+    _, ext = op.splitext(fname)
+    if ext == '.elp' or ext == '.hsp':
```

Comment:
So far I have not encountered any other order, and I assume you haven't either because this method of reading points in order is from you :) , I assume it is the order of acquisition. But yeah we could make the reader aware of the semantics of the *.elp file. I don't have an NY elp file here, are the points also called Mark1-Mark5?